lib/mix/tasks/ecto.migrations.ex

```diff
@@ -34,33 +34,30 @@ defmodule Mix.Tasks.Ecto.Migrations do
   def run(args, migrations \\ &Ecto.Migrator.migrations/1, puts \\ &IO.puts/1) do
     repos = parse_repo(args)
 
-    result = Enum.map(repos, fn repo ->
-      ensure_repo(repo, args)
-      ensure_migrations_path(repo)
-      {:ok, pid, _} = ensure_started(repo, all: true)
+    format = fn x, n -> x |> to_string() |> String.pad_trailing(n) end
 
-      repo_status = migrations.(repo)
+    result =
+      Enum.map(repos, fn repo ->
+        ensure_repo(repo, args)
+        ensure_migrations_path(repo)
+        {:ok, pid, _} = ensure_started(repo, all: true)
 
-      pid && repo.stop(pid)
+        repo_status = migrations.(repo)
 
-      """
+        pid && repo.stop(pid)
 
-      Repo: #{inspect repo}
+        """
 
-        Status    Migration ID    Migration Name
-      --------------------------------------------------
-      """ <>
-      Enum.map_join(repo_status, "\n", fn({status, number, description}) ->
-        status =
-          case status do
-            :up   -> "up  "
-            :down -> "down"
-          end
+        Repo: #{inspect(repo)}
 
-        "  #{status}      #{number}  #{description}"
-      end) <> "\n"
-    end)
+          Status    Migration ID    Migration Name
+        --------------------------------------------------
+        """ <>
+          Enum.map_join(repo_status, "\n", fn {status, number, description} ->
+            "  #{format.(status, 4)}      #{format.(number, 14)}  #{description}"
```

Comment:
Actually I can just increase the numbers here and remove all manual whitespaces, not sure which is better.